approval_polls/models.py

```diff
@@ -44,7 +46,30 @@ def voters(self):
     def __unicode__(self):
         return self.question
 
-
+    def can_edit(self):
+        return self.total_ballots == 0
```

Comment:
`self.total_ballots` is a method of Poll and hence is always returning False when compared to 0. You'd need to use `self.total_ballots()`